### PR TITLE
HW-297: Sets settings when installing Mosaico extension

### PR DESCRIPTION
**Pull request by meladawy**
Set default layout to 'bootstrap-wizard' using Mosaico settings file instead of using cv command

### CODE REVIEW

File: mosaico.php [hunk 1]
```diff
@@ -39,6 +39,7 @@ function mosaico_civicrm_install() {
  */
 function mosaico_civicrm_postInstall() {
   _mosaico_civix_civicrm_postInstall();
+  _mosaico_civicrm_install_settings();
```
Comment (omarabuhussein) [inline mosaico.php]: it is better to keep the order like this : 

```php
_mosaico_civicrm_install_settings();

_mosaico_civix_civicrm_postInstall();
```

**Review by omarabuhussein — CHANGES_REQUESTED** [review]

**Review by omarabuhussein — APPROVED** [review]

**Comment by guanhuan** [comment]
Looks good now. Please merge.

**Comment by totten** [comment]
IMHO, this PR was still incorrectly written. See point #2 from the previous discussion in https://github.com/veda-consulting/uk.co.vedaconsulting.mosaico/pull/146 -- the aim is for `auto` to stay pegged to mean "the default on new installations"

**Comment by meladawy** [comment]
Hey @totten, Sorry I missed this one as its got merged before we complete our discussion. I'll update this PR later today. Thanks!

**Pull request merged**